src/schema/resolvers/user.js

```diff
@@ -0,0 +1,9 @@
+import * as userController from '~/src/controllers/user'
+
+const resolvers = {
+  Query: {
+    currentUser: userController.currentUser
+  }
```

Comment:
I'd suggest adding a resolver for Groups, so that you don't have to include them *always*.
```javascript
const resolvers = {
￼  Query: {
￼    currentUser: userController.currentUser
￼  },
  User: {
    groups: userController.getGroups
  }
}
```